### PR TITLE
docs: add org scope to quickstart

### PR DESCRIPTION
**Pull request by hifabienne**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/guides/start/quickstart.mdx [hunk 1]
```diff
@@ -621,23 +621,14 @@ This tutorial covered how to configure ZITADEL and how to use React to build an
 We hope you enjoyed the tutorial and encourage you to check out the ZITADEL [documentation](https://zitadel.com/docs) for more information on how to use the ZITADEL platform to its full potential. Thanks for joining us!
 
 
+## Show Organization Login
```
Comment (dakshitha) [inline docs/docs/guides/start/quickstart.mdx:624]: This part won't fit here because we do not cover audiences.

**Review by dakshitha — CHANGES_REQUESTED** [review]
Please see above comments.

**Comment by hifabienne** [comment]
> https://zitadel.com/docs/guides/manage/console/organizations

Yes we can try that. Not sure if it is enough.

**Comment by hifabienne** [comment]
I have changed it

**Review by dakshitha — APPROVED** [review]

**Pull request merged**